### DISCUSSION TITLE
Wallet Connect request Issue does not redirect to sign request

### DISCUSSION THREAD

**Discussion by aqibnoborderz** [WalletConnect/walletconnect-monorepo#4300] [Developer Support]
When i genereate sign request using following code it do not reidrect to metamask and the internal api that walletconnect SDK calls is failing everytime.

code :
 const web3 = new Web3(library.provider as any)
                const signature = await web3?.eth?.personal?.sign(
                    signMessageToken,
                    account,
                    '',
                    () => {},
                )

**Comment by glitch-txs** [comment]
Hi, could you share a minimal reproducible example and steps to reproduce the issue? Thanks!
